devito/operator.py

```diff
@@ -602,8 +606,24 @@ def _profile_output(self, args):
                 perf("%s* %s%s computed in %.2f s"
                      % (indent, name, rank, fround(v.time)))
 
+        # Emit relevant configuration values
         perf("Configuration:  %s" % self._state['optimizations'])
 
+        # Emit relevant performance arguments
+        perf_args = {}
+        for i in self.input + self.dimensions:
+            if not i.is_PerfKnob:
+                continue
+            try:
+                perf_args[i.name] = args[i.name]
+            except KeyError:
+                # Try with the aliases
+                for a in i._arg_names:
+                    if a in args:
+                        perf_args[a] = args[a]
+                        break
+        perf("Performance arguments:  %s" % perf_args)
```

Comment:
with a low log level, now also emitting the performance-related arguments values when returning from op.apply

E.g.
```
...
Performance arguments:  {'nthreads_nonaffine': 4, 'nthreads': 4, 'x0_blk0_size': 8, 'y0_blk0_size': 8}
```

Reply:
`Perf` is stil lbroken I think so not sure this is usefull.